shared/modules/error-utils.js

```diff
@@ -0,0 +1,46 @@
+import getFirstPreferredLangCode from '../../app/scripts/lib/get-first-preferred-lang-code';
+import { setupLocale } from '../../ui';
+
+let currentLocaleMessages, enLocaleMessages;
+
+getFirstPreferredLangCode().then((preferredLocale) => {
```

Comment:
Generally we should not be using `.then` anymore, because async stack traces don't work with `.then`. Instead move this into an async function and use `await`.

Also, the error case isn't handled here. What should we do if this fails? When would we expect this to fail? This goes for both operations.

Right now, if this fails, it would bubble up and get sent to Sentry. It would also cause each invocation of the `t` function below to throw an exception.